src/tools/visualizer/index.js

```diff
@@ -81,6 +81,9 @@ const visualizer = React.createClass({
       ImageProviders.setImageProvider(null);
     });
     dispatch(Actions.resetVisualizerState());
+
+    // Close ws without exiting server
+    network.exit(-1);
   },
```

Comment:
I had to push some changes in paraviewweb for it yesterday, but now that we are using the latest Visualizer and ParaViewWeb we are good.